src/precice/impl/Participant.hpp

```diff
@@ -83,10 +83,10 @@ class Participant {
       int                  interpolationOrder);
 
   /// Adds a configured read \ref Mapping to the Participant
-  void addReadMappingContext(MappingContext *mappingContext);
+  void addReadMappingContext(MappingContext mappingContext);
 
   /// Adds a configured write \ref Mapping to the Participant
-  void addWriteMappingContext(MappingContext *mappingContext);
+  void addWriteMappingContext(MappingContext mappingContext);
```

Comment:
Wouldn't it make sense to use references here as well?

Reply:
Yes indeed, will change it to const reference.